src/tests/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.AcceptanceTests/Features/EmploymentCheck.feature

```diff
@@ -11,7 +11,6 @@ Scenario: Employment checks requested on first ILR submission
 	And a 365 employment check is not requested
 Examples:
 	| Phase  |
```

Comment:
Phase is the column heading. Phase1 has been removed from the table because employment checks are not requested for those applications, due to the timescales